unstructured/partition/email.py

```diff
@@ -101,11 +176,20 @@ def partition_email(
     #    <li>Item 1</li>=
     #    <li>Item 2<li>=
     # </ul>
-    content = "".join(content.split("=\n"))
+    list_content = split_by_paragraph(content)
 
-    elements = partition_html(text=content)
-    for element in elements:
-        if isinstance(element, Text):
-            element.apply(replace_mime_encodings)
+    if content_source == "text/html":
+        content = "".join(list_content)
+        elements = partition_html(text=content)
 
-    return elements
+        for element in elements:
+            if isinstance(element, Text):
+                element.apply(replace_mime_encodings)
+    elif content_source == "text/plain":
+        elements = partition_text(file_content=list_content)
+
+    header: List[Union[Text, Name, List[Name]]] = list()
+    if get_meta_data:
+        header = partition_header(msg)
+
+    return (elements, header)
```

Comment:
I'd rather keep this returning `List[Elements]` to keep the `partition_email` interface the same as the other `partition` functions. Are the `partition_header` outputs `Element` objects? If so, what about making the kwarg `include_headers` and adding them to the top of the list if `include_headers` is `True`? That would keep the interface the same.